freedomofpress/securedrop
# Provide 0.12.2 upgrade boxes

## Status

Work in progress : we need to finish uploading historical boxes to the new s3 bucket and update URLs in this PR

## Description of Changes

Closes #4338, closes #4329.


Changes proposed in this pull request:

## Testing

- checkout this branch
- run `make build-debs`
- [ ] `make upgrade-start` pulls in the newly-built 0.12.2 box (see Source Interface)
- [ ] `make upgrade-test-local` upgrades the box to 0.13.0~rc1 without error (see Source Interface)
- [ ] `make upgrade-trusty-start` pulls in the newly-built 0.12.2 box (see Source Interface)
## Deployment

Developer environment only

## Checklist


### If you made non-trivial code changes:
